roles/openshift_node_group/tasks/sync.yml

```diff
@@ -83,8 +83,9 @@
     - node_status.results.results is defined
     - node_status.results.results | length > 0
     - node_status.results.results[0]['items']
-     | map(attribute='metadata.annotations') | map('list') | flatten
-     | select('match', '[\"node.openshift.io/md5sum\"]') | list | length == node_status.results.results | length
```

Comment:
`node_status.results.results | length` would always be 1, as `oc_obj` returns a single List object.